CMakeLists.txt

```diff
@@ -139,16 +139,16 @@ if(DEFINED EUDAQ_TTREE_LIBRARY)
   list(APPEND CONF_EUDAQ_LIBRARIES eudaq_${EUDAQ_TTREE_LIBRARY})
 endif()
 
-set(CONF_INCLUDE_DIRS "\${EUDAQ_INSTALL_DIR}/include")
+set(CONF_INCLUDE_DIRS "\${EUDAQ_INSTALL_DIR}/include \${EUDAQ_INSTALL_DIR}/extern/include")
```

Comment:
the external dependency, mainly lcio is installed at extern/include.   This path is reqiured when build LCIO converter by import eudaqTargets.cmake